actions/log_action/pubspec_overrides.yaml

```diff
@@ -0,0 +1,7 @@
+dependency_overrides:
+  fluttium:
+    path: /Users/wolfen/Projects/personal/fluttium/packages/fluttium
```

Comment:
Do you want to track these files?

Reply:
oh this shouldn't be absolute, should be relative. But they should be tracked to make contributing easier. They are ignored by pub publish afaik